### PR TITLE
Code review request

### PR DESCRIPTION
**Pull request by Wintertown**
@lparker7, check formatting, naming and logic. Namely:

- No global variables
- Meaningful variable names
- Line length
- Understandable logic flow

### CODE REVIEW

File: prac_05/word_count.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+def print_dictionary(dictionary):
+    longest_key_length = max([len(key) for key in dictionary])
+    for key in dictionary:
+        print("{:{}} : {}".format(key, longest_key_length, dictionary[key]))
+
+
+text = input("Text: ").split()
+text.sort()
+words = {}
```
Comment (LParker7) [inline prac_05/word_count.py]: Words may be an inappropriate variable name, considering its purpose is 'counting' word occurrences

File: prac_05/colour_names.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+                "NavajoWhite": "#ffdead", "MediumAquamarine": "#66cdaa", "CadetBlue": "#5f9ea0",
+                "DodgerBlue": "#1e90ff", "Thistle": "#d8bfd8", "Honeydew": "#fff0ff", "Gainsboro": "#dcdcdc"}
+# print(COLOUR_NAMES)
+
+colour = input("Enter X11 colour name: ")
+while colour != "":
+    if colour in COLOUR_NAMES:
+        print(colour, "is", COLOUR_NAMES[colour])
+    else:
+        print("Invalid colour name")
```
Comment (LParker7) [inline prac_05/colour_names.py]: This error message might be misleading. What if their colour simply isn't in your colour dictionary?

File: prac_05/state_names.py [hunk 3]
```diff
@@ -1,18 +1,22 @@
...
     if state in STATE_NAMES:
         print(state, "is", STATE_NAMES[state])
     else:
         print("Invalid short state")
     state = input("Enter short state: ")
-    
\ No newline at end of file
+
+
+def print_dictionary(dictionary):
```
Comment (LParker7) [inline prac_05/state_names.py]: The program does not allow for the use of this function. Also, if using a separate function for this task, would it be more appropriate to put the rest of the code in a 'main' function?

File: prac_05/word_count.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+for word in text:
+    if word in words:
+        words[word] += 1
+    else:
+        words[word] = 1
+print_dictionary(words)
```
Comment (LParker7) [inline prac_05/word_count.py]: Will printing the dictionary guarantee alphabetical order?

File: prac_05/word_count.py (hunk 1 above)
Comment (Wintertown) [reply prac_05/word_count.py]: Updated to use `word_counts`

File: prac_05/colour_names.py (hunk 2 above)
Comment (Wintertown) [reply prac_05/colour_names.py]: Changed to 'unsupported', which is true for cases where the name is invalid or not included.

File: prac_05/state_names.py (hunk 3 above)
Comment (Wintertown) [reply prac_05/state_names.py]: Refactored `print_dictionary` into its own file.

File: prac_05/word_count.py (hunk 4 above)
Comment (Wintertown) [reply prac_05/word_count.py]: Replaced with `print_sorted_dictionary(dictionary)`, which guarantees alphabetical order when printed.

**Pull request merged**